nuxeo-drive-client/nxdrive/utils.py

```diff
@@ -297,17 +297,16 @@ def find_resource_dir(directory, default_path):
     return dir_path
 
 
-def force_decode(string, codecs=['utf-8', 'cp1252']):
+def force_decode(string, codecs=('utf-8', 'cp1252')):
```

Comment:
It is a matter of security. It is dangourous to use `dict` as a default value because it is not immutable and functions arguments are evaluated at compile time, not at runtime. For instance:
``` python
>>> def force_encoding(text, codecs=['utf-8', 'cp1252']):
...    print(codecs)
...    codecs.append('oups!')
>>> force_encoding('foo')
['utf-8', 'cp1252']
>>> force_encoding('bar')
['utf-8', 'cp1252', 'oups!']


>>> def force_encoding_secure(text, codecs=('utf-8', 'cp1252')):
...    print(codecs)
...    codecs += tuple(['oups!'])
>>> force_encoding_secure('foo')
('utf-8', 'cp1252')
>>> force_encoding_secure('bar')
('utf-8', 'cp1252')
```